### PR TITLE
fix(policies): Splitting interfaces with wildcards

### PR DESCRIPTION
**Pull request by ekhandigi**
When creating a policy firewalld does not handle wildcard interfaces
(i.e. tun+). These interfaces need to be dealt with differently due to
the fact nftables cannot parse them in a 'set' (i.e. it will not
register a wildcard if given {"tun*", "net1"}).

This have been overcome through identifying the interfaces with
wildcards and splitting them out.

Before:
ifname { "eth0", "eth1" "tun+" } oifname { "eth1", "eth2" } jump filter_FWD_policy_test

After:
ifname "tun*" oifname { "eth1", "eth2" } jump filter_FWD_policy_test
ifname { "eth0", "eth1" } oifname { "eth1", "eth2" } jump filter_FWD_policy_test

### CODE REVIEW

**Comment by erig0** [comment]
Hi, thanks for the fix and the associated test case!

I have not reviewed this yet, but I'm also making changes in this area to address #797. Assuming review is okay then I'll take your change before mine for #797. It will be good to have your test case. :)

**Comment by AddisonG** [comment]
Looks good to me. Waiting on this fix.

**Comment by erig0** [comment]
Hi, sorry for the delay. I got busy and this fell off my radar. Anyways, I rebased it and simplified the code a bit. Just waiting on CI before I merge.

Thanks!

**Review by erig0 — APPROVED** [review]

**Pull request merged**
